dandi/bids_validator_xs.py

```diff
@@ -450,7 +452,7 @@ def validate_all(
 
 def write_report(
     validation_result,
-    report_path="/var/tmp/bids-validator-report_{}.log",
+    report_path="{logdir}/bids-validator-report_{date}.log",
```

Comment:
do not use `/` but `os.sep` to provide OS agnostic one?

To say the truth -- I do not see really a reason for all this automated saving into some generic directory which is disconnected from the dataset where validation is done.  Imagine that `ls` by default would have `ls`ed content into some global location.

If you were to be fancy, I would have also defined some `dataset` path variable (but it isn't known, would need to be passed).  But then again -- why bother since outside code could easily provide it.

Also even with a detailed datetime to a second, collision could happen and thus default one better has some process ID . Altogether then may be make it by default the same as we have for logs

```
dandi/cli/command.py:        logdir, "{:%Y%m%d%H%M%SZ}-{}.log".format(datetime.utcnow(), os.getpid())
```
which includes that PID and explicit time zone (or lack of such).

And since it is `datetime`, call it such instead of `date`  to be more factual.